domains/dyikes.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "dyikes",
+        "email": "f@power.slmail.me"
+    },
+    "record": {
+        "CNAME": "z.dyikes.cloudns.be"
```

Comment:
you should complete your website first, i only see "This website is working properly!"